fenix/app/src/main/java/org/mozilla/fenix/onboarding/JunoOnboardingFragment.kt

```diff
@@ -43,9 +43,11 @@ class JunoOnboardingFragment : Fragment() {
     private val telemetryRecorder by lazy { JunoOnboardingTelemetryRecorder() }
 
     @SuppressLint("SourceLockedOrientationActivity")
-    override fun onCreate(savedInstanceState: Bundle?) {
-        super.onCreate(savedInstanceState)
-        activity?.requestedOrientation = ActivityInfo.SCREEN_ORIENTATION_PORTRAIT
+    override fun onStart() {
```

Comment:
I am just wondering about the change from `onCreate` to `onStart` similarly for the changes below. I am see that we currently set this in `onCreate` everywhere https://searchfox.org/mozilla-mobile/search?q=SCREEN_ORIENTATION_PORTRAIT&path=. So, I am wondering what is the best practise here or reasoning around the change?

Reply:
This is a separate bug fix meant for BZ-1827294 which has slipped into this change. @rahulsainani could you separate this into a separate commit? Keeping it in the same PR is fine.

Reply:
This was done as a preemptive "fix" for https://bugzilla.mozilla.org/show_bug.cgi?id=1827294 to basically set and reset `requestedOrientation`.

Reply:
Alright, thanks!

Reply:
I got a chance to test out this changeset (specifically 014a57fbb625e9816098c40e48e22fac10455fff) on my device and I'm still able to see the bug. With the whole changeset of course, this bug isn't present because we are now choosing not to change orientation for tablets.

My guess might have been wrong about the calls and maybe it has something to do with the screen orientation value we choose? Does unspecified mean that it's supposed to go back to the previous value? Should it be [`SCREEN_ORIENTATION_SENSOR`](https://developer.android.com/reference/android/content/pm/ActivityInfo#SCREEN_ORIENTATION_SENSOR) or [`SCREEN_ORIENTATION_USER`](https://developer.android.com/reference/android/content/pm/ActivityInfo#SCREEN_ORIENTATION_USER)? I'm not really sure.

Reply:
I think if you're looking to land this soon, then either:
1. Timebox an investigation for what is the appropriate fix - using a different orientation value, caching the previous orientation value, etc. and documenting it inline or in the bug.
3. Removing the fix for BZ-1827294 from this change.

Reply:
Thanks for testing! Yes, I was wondering about those orientations looking in the docs as well. Since I cannot reproduce it on the devices I have, it would be hard to find a solution without requesting you to test it. For now, I've removed the fix for [BZ-1827294](https://bugzilla.mozilla.org/show_bug.cgi?id=1827294) from this PR, so we can review/land supporting both orientations on tablet.

Reply:
That sounds good. Let's land this then! 🙂